src/QMCHamiltonians/BareForce.h

```diff
@@ -0,0 +1,58 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: John R. Gergely,  University of Illinois at Urbana-Champaign
+//                    Ken Esler, kpesler@gmail.com, University of Illinois at Urbana-Champaign
+//                    Jeongnim Kim, jeongnim.kim@gmail.com, University of Illinois at Urbana-Champaign
+//                    Jeremy McMinnis, jmcminis@gmail.com, University of Illinois at Urbana-Champaign
+//                    Raymond Clay III, j.k.rofling@gmail.com, Lawrence Livermore National Laboratory
+//                    Mark A. Berrill, berrillma@ornl.gov, Oak Ridge National Laboratory
+//
+// File created by: John R. Gergely,  University of Illinois at Urbana-Champaign
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_BARE_FORCE_HAMILTONIAN_H
+#define QMCPLUSPLUS_BARE_FORCE_HAMILTONIAN_H
+#include "QMCHamiltonians/ForceBase.h"
+#include "QMCHamiltonians/OperatorBase.h"
+
+namespace qmcplusplus
+{
+struct BareForce : public OperatorBase, public ForceBase
+{
+public:
+  BareForce(ParticleSet& ions, ParticleSet& elns);
+  std::string getClassName() const override;
```

Comment:
For these simple getters and setters I tend to use `noexcept` , but this is an inherited function as it could be part of future PRs.

Reply:
Are you saying you would add `noexcept` even though it's a virtual function override?

Reply:
That adding `noexcept` would need to be done at the base class level so it will propagate, but in a future PR.